rticonnextdds_connector/rticonnextdds_connector.py

```diff
@@ -544,101 +592,124 @@ class Instance:
 		An Instance has an associated DDS Type, specified in the XML configuration,
 		and it allows setting the values for the fields of the DDS Type.
 
-		TODO: fieldName format (create new section and link to it)
-
 		Attributes:
 			* ``output`` (:class:`Output`): The ``Output`` that owns this ``Instance``.
+
+		Special methods:
+			* ``__setitem__``, see `:ref:`Accessing the data`.
 	"""
 
 	def __init__(self, output):
 		self.output = output
 
-	def clear_member(self, fieldName):
+	def clear_member(self, field_name):
 		"""Resets a member to its default value
 
 		The effect is the same as that of :meth:`Output.clear_members()` except
 		that only one member is cleared.
 
-		:param str fieldName: The name of the field. It can be a complex member or a primitive member. See :ref:`Accessing the data`.
+		:param str field_name: The name of the field. It can be a complex member or a primitive member. See :ref:`Accessing the data`.
 		"""
 
 		retcode = rtin_RTIDDSConnector_clearMember(
 			self.output.connector.native,
 			tocstring(self.output.name),
-			tocstring(fieldName))
+			tocstring(field_name))
 		_check_retcode(retcode)
 
-	def set_number(self, fieldName, value):
+	def __setitem__(self, field_name, value):
+		"""Sets the value of field_name
+
+		:param str field_name: The name of the field. See :ref:`Accessing the data`.
+		:param number value: A numeric, boolean or string value or ``None`` to unset an optional member.
+
+		The type of the argument must correspond with the type of the field as defined
+		in the configuration file.
+
+		This is an alternative to :meth:`set_number`, :meth:`set_string` and :meth:`set_boolean`
+		"""
+		if isinstance(value, Number):
```

Comment:
is_instance?

Reply:
It's a python bulit-in function called `isinstance` https://docs.python.org/3/library/functions.html#isinstance